src/pages/Vaults.tsx

```diff
@@ -117,18 +136,21 @@ function processVaultsData(
       throw new Error(`Node ID does not contain enough segments: ${node.id}`);
     }
     const managerName = idSegments.slice(0, 4).join('.');
-    const {liquidatingCollateralBrand} = node;
+
+    const liquidatingCollateralBrand = node.liquidatingCollateralBrand;
+    const decimalPlaces = (boardAuxes && boardAuxes[liquidatingCollateralBrand]) || 6;
```

Comment:
keeping a default value of 6 here could potentially obscure any issue on the frontend.
we might have a missing boardAux and due to this safety check we might be using another value instead

something like 0 might be a better decision to use here?

Reply:
Perhaps. But at the moment, we only get `boardAuxes` data for `ATOM`. For now, the fallback of `6` works for tokens whose data is not fetched by `boardAuxes` query.